internal/querycoordv2/session/cluster.go

```diff
@@ -140,6 +149,10 @@ func (c *QueryCluster) GetDataDistribution(ctx context.Context, nodeID int64, re
 	var resp *querypb.GetDataDistributionResponse
 	var err error
 	err1 := c.send(ctx, nodeID, func(cli *grpcquerynodeclient.Client) {
+		req := proto.Clone(req).(*querypb.GetDataDistributionRequest)
+		req.Base = &commonpb.MsgBase{
```

Comment:
req.Base.TargetID = nodeID?

Reply:
`req.Base` is nil here

Reply:
Create `req.Base` outside and only modifying `TargetID` here?

Reply:
Fixed